devsetup/roles/download_tools/defaults/main.yaml

```diff
@@ -7,10 +7,10 @@ opm_version: latest
 
 # operator-sdk version to use (must be specific version)
 #sdk_version: v0.19.2 - cnosp is right now based on that version
-sdk_version: v1.14.0
+sdk_version: v1.26.0
 
 # golang version
-go_version: 1.16.9
+go_version: 1.19.0
```

Comment:
Should we have this match what is in `vars/default.yaml`?  I know the `vars` setting actually "wins", but perhaps they should be identical.

Reply:
We probably should not have it multiple places, but updated.